src/main/java/com/google/sps/data/ScheduledInterviewDao.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/**
+ * ScheduledInterviewDao includes the basic methods anything managing ScheduledInterview entities
+ * must support.
+ */
+public interface ScheduledInterviewDao {
+  // Returns a ScheduledInterview object from storage
+  public ScheduledInterview get() throws com.google.appengine.api.datastore.EntityNotFoundException;
+
+  // Puts a ScheduledInterview object into storage.
+  public void put(ScheduledInterview scheduledInterview);
```

Comment:
Consider having create() and update() methods rather than just put. I think this generally keeps the logic cleaner both inside the methods and at the call sites. You don't necessarily have to do this, but I'd like to hear your reasoning if you don't.

Reply:
I meant that you should also delete `put` here since it is no longer necessary with `create` and `delete`. Sorry, I see why this wasn't clear from my first comment.

Reply:
Did you mean create and update here?